### PR TITLE
Add PR template, dynver, drafter, scala steward

### PR DESCRIPTION
**Pull request by pperzyna**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,15 +1,12 @@
 logLevel := Level.Warn
 
-resolvers += Resolver.typesafeRepo("releases")
+resolvers += "FrugalMechanic Snapshots" at "s3://maven.frugalmechanic.com/snapshots"
```
Comment (xavji) [inline project/plugins.sbt:3]: you should not need that resolver

File: build.sbt [hunk 2]
```diff
@@ -1,10 +1,21 @@
 import Dependencies.Library
 
+lazy val commonSettings = Seq(
+  publishMavenStyle := true,
+  publishTo := {
+    val prefix = if (isSnapshot.value) "snapshots" else "releases"
+    Some(s"Models$prefix" at s"s3://library-artifacts-$prefix.hubofallthings.com")
```
Comment (benshaw) [inline build.sbt:7]: Models should be replaced with "HAT Library Artifacts" as this is what it was previously

File: project/plugins.sbt (hunk 1 above)
Comment (pperzyna) [reply project/plugins.sbt]: done

File: build.sbt (hunk 2 above)
Comment (pperzyna) [reply build.sbt]: done

**Review by xavji — APPROVED** [review]

**Pull request merged**